ecllibrary/std/Crypto.ecl

```diff
@@ -84,17 +84,43 @@ EXPORT SymmetricEncryption(VARSTRING algorithm, VARSTRING passphrase) := MODULE
     END;
 END; // SymmetricEncryption module
 
-
-
+/**
+ * Encryption module containing symmetric encryption/decryption functions
+ *
+ * @param   algorithm      Symmetric algorithm to use, as returned by SupportedSymmetricCipherAlgorithms()
+ * @param   passphrase     Passphrase buffer to use for encryption/encryption
+ */
+EXPORT SymmEncryption(VARSTRING algorithm, DATA passphraseBuff) := MODULE
```

Comment:
The external function changes are required, but I'm just wondering if this is necessary.  What happens if you change the parameter type for the original module.  Does all of the existing code still work?  My initial suspicion is that it would.

If it did work then it reduces the complication for the user, and the documentation overhead.

Reply:
Legacy code generates a compile error if VARSTING changed to DATA